src/Features/Core/Portable/NavigateTo/INavigateToSearchCallback.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.NavigateTo;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.NavigateTo
+{
+    internal interface INavigateToSearchCallback
```

Comment:
created an interface similar to the editor interface that I could use in the features layer and LSP layer.